sys-fs/bees/bees-9999.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit multilib linux-info
+
+DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
+HOMEPAGE="https://github.com/Zygo/bees"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/kakra/bees.git"
+	EGIT_BRANCH="integration"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Zygo/bees/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+PATCHES=(
+	"${FILESDIR}/v0.5-gentoo_build.patch"
+)
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="tools test"
+
+RDEPEND=">=sys-apps/util-linux-2.30.2"
+DEPEND="$RDEPEND >=sys-fs/btrfs-progs-4.1"
+
+DOCS="README.md COPYING"
```

Comment:
We don't install `COPYING` as it's covered by `LICENSES` in ebuild. Then, `README*` default should cover it, so you can remove `DOCS` override completely.

Reply:
I didn't know of this default, fixed.